test-network-function/identifiers/identifiers.go

```diff
@@ -333,6 +337,19 @@ test case requires the Deployment of the debug daemonset.
 		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
 	},
 
+	TestICMPv4ConnectivityIdentifierMultus: {
+		Identifier: TestICMPv4ConnectivityIdentifierMultus,
+		Type:       normativeResult,
+		Remediation: `Ensure that the CNF is able to communicate via the Multus network(s). In some rare cases,
+CNFs may require routing table changes in order to communicate over the Multus network(s). For instructions on
+how to exclude a particular container from ICMPv4 connectivity tests, consult:
+[README.md](https://github.com/test-network-function/test-network-function#issue-161-some-containers-under-test-do-not-contain-ping-or-ip-binary-utilities).`,
```

Comment:
Bad link. We've removed this from README.

Reply:
We still have one sentence there mentioning the exclude label. Not sure we need to link that or maybe just mention it here too.

Reply:
I'll just add the text, I think it makes sense. Thanks!